FordLabs/PeopleMover
# Include space name in header

## Issue
Connects #375

## What was done
- [x] Added the space name to the header when on a space page
- [x] Componentized the `PEOPLEMOVER` logo and text
- [x] Made it so that the people mover logo is not clickable on the dashboard but is when in a space
- [x] Changed all the classes in the LandingPage from dashed classes to camel case.

## How to test
1. Ensure the Login page logo looks the same
2. Ensure the logo is *not* clickable when on the dashboard
3. Go to a space and ensure the people mover logo is clickable and leads back to the dashboard, AND that you can now see the space name after the logo (and that it matches the figma)
